app.js

```diff
@@ -2,26 +2,52 @@
 
 const express = require('express');
 const app = express();
+const gapi = require('./domains');
```

Comment:
**A1** + **A2**: I was thinking the same thing, but if `domains.js` is going to be deleted, I don't think it's worth it to have `app.js` in a folder by itself and worry about changing paths in other files to accommodate that. Can we prefix `domains.js` as `tempfix_domains.js`?